react-components/src/components/AxisGizmo.tsx

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+import { type ReactNode, useEffect } from 'react';
+
+import { AxisGizmoOptions, AxisGizmoTool } from '@cognite/reveal/tools';
+import { useReveal } from '..';
+
+export { AxisGizmoOptions };
+
+/**
+ * A React wrapper around the AxisGizmoTool from Reveal
+ */
+export const AxisGizmo = ({ options }: { options?: AxisGizmoOptions }): ReactNode => {
```

Comment:
I prefer to use `ReactElement`, any specific reasoning to use `ReactNode` here?

Reply:
No - I can change it